vtex/faststore
# Feat: Adds Missing Components on FS/Core

## What's the purpose of this pull request?
This PR adds missing components on `fs/core`:
- [x] ProductCard
- [x] PaymentMethods

## How it works?
It should all works as before (visually).

## How to test it?
Run `yarn dev` and check `ProductCard` and `PaymentMethods` (footer) components on the store.
